hack/util.sh

```diff
@@ -127,6 +127,21 @@ function configure_os_server {
 	echo "[INFO] To debug: export KUBECONFIG=$ADMIN_KUBECONFIG"
 }
 
+# ensure_no_openshift displays existing OS processes.  Existing processes may cause
```

Comment:
If we're going to keep this in `test-cmd`, s/ensure_no_openshift/check_for_running_openshift/